modules/edu.gemini.seqexec.engine/src/main/scala/edu/gemini/seqexec/engine/Sequence.scala

```diff
@@ -138,6 +142,10 @@ object Sequence {
 
     def getCurrentBreakpoint: Boolean
 
+    def setObserver(name: String): State
```

Comment:
Yes, it sets the same operator name for all the loaded Sequences. If you want the operator name in a single place, let me know where to put it because as I mentioned above, it's not obvious to me where to put it.